test/integration/application.spec.ts

```diff
@@ -28,7 +28,7 @@ describe('Application', () => {
   it('should allow GET /', async () => {
     const result = await request(server).get('/');
     expect(result.status).to.equal(HttpStatus.OK);
-    expect(result.text).to.equal('home');
+    expect(result.notFound).to.be.false;
```

Comment:
What is setting `result.notfound`? I don't see it in the `supertest` docs.

Reply:
I guess I'm not clear on why we are setting `notFound` at all. The HTTP status could be 404 when not found and 200 when OK. This property feels redundant and uneccary in a success (200 OK) case.